rosidl_generator_py/cmake/rosidl_generator_py_generate_interfaces.cmake

```
@@ -301,5 +301,10 @@ if(BUILD_TESTING AND rosidl_generate_interfaces_ADD_LINTER_TESTS)
       # a value of zero tells uncrustify to ignore line length
       MAX_LINE_LENGTH 0
       "${_output_path}")
+
+    find_package(ament_cmake_mypy REQUIRED)
+    ament_mypy(
+      TESTNAME "mypy_rosidl_generate_py"
```

To match the other `*_rosidl_generated_py` test names:

```suggestion
      TESTNAME "mypy_rosidl_generated_py"
```